db/data_migration/20160309153708_republish_organisations.rb

```diff
@@ -0,0 +1 @@
+Organisation.all.each(&:publish_to_publishing_api)
```

Comment:
Should use the republisher now for this: https://github.com/alphagov/whitehall/blob/master/lib/data_hygiene/publishing_api_republisher.rb. See https://github.com/alphagov/whitehall/blob/master/db/data_migration/20160219145627_republish_working_groups.rb for an example.